src/helm/proxy/clients/auto_client.py

```diff
@@ -263,8 +263,19 @@ def get_critique_client(self) -> CritiqueClient:
             if not surgeai_credentials:
                 raise ValueError("surgeaiApiKey credentials are required for SurgeAICritiqueClient")
             self.critique_client = SurgeAICritiqueClient(surgeai_credentials, self._build_cache_config("surgeai"))
+
+        elif critique_type == "scale":
+            scale_credentials = self.credentials.get("scaleApiKey")
+            scale_project = self.credentials.get("scaleProject", None)
```

Comment:
Should not need `scaleProject`:

https://docs.scale.com/reference/text-collection

"Note that if a batch is specified, you need not specify the project, as the task will automatically be associated with the batch's project."